### DISCUSSION TITLE
How to choose cypress component destination

### DISCUSSION THREAD

**Discussion by MatthD** [cypress-io/cypress#23560] [Component Testing]
Hey guys .

I am facing multiple issues (jest/cypress types collision) that lead me to put my cypress tests in the cypress folder and not the component directory of my `nuxt2 app` BUT I want to be able to use the GUI to generate spec test component , 
It works but it put me the test file next to the component, so I have to move the file and correct paths then.

Is there a way to indicate to put test elsewhere?

**Comment by emilyrohrbough** [comment]
@MatthD You can update your specPattern to include the path to your component testing folder to correct this path. For example:
```diff
  component: {
+   specPattern: '/cypress/component/hello/**.cy.js',
    devServer: {
      framework: 'nuxt',
      bundler: 'webpack'
    }
  }
```
